content/docs/studio/how-it-works.md

```diff
@@ -1,48 +1,54 @@
 # How Iterative Studio Works
 
-[Iterative Studio](https://studio.iterative.ai/) works with the data, metrics,
-hyperparameters and model metadata that you add to your ML project Git
-repositories. It works very closely with your Git ecosystem.
+[Iterative Studio](https://studio.iterative.ai/) works very closely with your
+Git ecosystem. It parses the data, metrics, hyperparameters and model metadata
+that you add to your ML project Git repositories, and renders them on a web UI.
+It also parses and displays live metrics and plots sent by [DVCLive]; these
+don't need to be pushed to the Git repositories.
 
-This video illustrates how Iterative Studio works closely with your Git
-ecosystem.
+The following video illustrate how Iterative Studio works with data from your ML
+projects and experiments.
 
 > Note that we have renamed DVC Studio to Iterative Studio and Views to
 > Projects.
 
 https://www.youtube.com/watch?v=5xM5az78Lrg
 
-## How your project data is saved
+## How you save your project data
 
 - Using [DVC] and [Git], you will push all your ML experiments to your GitHub,
   GitLab or Bitbucket repositories as Git commits.
 - Using Iterative Studio, or using the command line interface (CLI) of [GTO],
   and possibly [MLEM], you will push all your ML model details to the Git
   repositories as Git commits and Git tags.
+- Using [DVCLive], you will push live metrics and plots from your Git-committed
+  experiments to Iterative Studio.
 
 ## How Iterative Studio extracts your project data
 
-- When you connect to these Git repositories from Iterative Studio, the
-  project's `dvc.yaml` is used to identify all the data, metrics and
-  hyperparameters in your experiments.
+- When you connect to the Git repositories from Iterative Studio, the project's
+  `dvc.yaml` is used to identify all the data, metrics and hyperparameters in
+  your experiments.
 - If you are not using DVC, you can
   [add the metrics and hyperparameters to your Git repositories manually](/doc/studio/user-guide/projects-and-experiments/configure-a-project#custom-metrics-and-parameters).
 - Details of your ML models, including versions and stage assignments, are
   extracted from the Git commits and tags.
+- Live metrics and plots for experiments committed to Git are extracted from the
+  data you send using [DVCLive].
```

Comment:
Wait. I thought live metrics/plots were not committed/pushed to Git? The mechanics start getting confusing here.

Reply:
The live metrics/plots aren't committed to Git. But the experiments are (there has to be a Git commit that triggers the experiment).

Reply:
IMO, `remote` is not enough, coz it does not imply that there is a Git commit for it. Users could interpret `remote` to mean any (cloud) machine that is not your laptop/desktop.

Reply:
I see 2 issues here:

- Emphasize where experiments run/are saved (local vs remote) and what that means to the user from the PoV of the Studio UI
- Differentiate (or not) between experiments and commits (long-standing DVC discussion). I incline to call the first "experiments" and the latter "commits" or "project versions" (even if we clarify that they used to be exps.

Reply:
@jorgeorpinel We've now already merged the docs for live metrics for local experiments. Can you pls read that version and comment if you still see issues?

Reply:
The only part that confuses me a little now is under https://dvc.org/doc/studio/how-it-works#how-iterative-studio-runs-experiments-and-model-actions:

> ### How Iterative Studio runs experiments...
>
> - When you [run new experiments](https://dvc.org/doc/studio/user-guide/projects-and-experiments/run-experiments) ... in Iterative Studio, it creates Git commits

So are experiments commits? (Could be what a reader wonders.)

Reply:
Yes, the experiments are pushed to Git, as Git commits and possibly PRs (if the change was submitted to a new branch).